### DISCUSSION TITLE
Different song in multi osu!lazer (round robin)

### DISCUSSION THREAD

**Discussion by Veldora2k** [ppy/osu#15945] [Q&A] [answered]
Hi. Easy to reproduce bug, 100% of time.
-> Press ready as player then press "add map" (to poll)
-> Browse maps, don't add, stay at this list at different than picked to play song
-> Host is starting match
-> You are playing last saw song on list, not this one that others are playing.

It's easy to reproduce, tried this couple times and it works every time.

**Comment by bdach** [comment, marked as answer]
#15946
